.evergreen/config.yml

```diff
@@ -2322,17 +2323,19 @@ axes:
           skip_web_identity_auth_test: true
           python3_binary: /Library/Frameworks/Python.framework/Versions/3.8/bin/python3
           libmongocrypt_url: https://s3.amazonaws.com/mciuploads/libmongocrypt/macos/master/latest/libmongocrypt.tar.gz
-      - id: rhel76
-        display_name: "RHEL 7.6"
+      - id: rhel7
+        display_name: "RHEL 7.x"
         run_on: rhel76-small
         batchtime: 10080  # 7 days
         variables:
+          python3_binary: "/opt/python/3.8/bin/python3"
           libmongocrypt_url: https://s3.amazonaws.com/mciuploads/libmongocrypt/rhel-70-64-bit/master/latest/libmongocrypt.tar.gz
-      - id: rhel84
-        display_name: "RHEL 8.4"
+      - id: rhel8
+        display_name: "RHEL 8.x"
```

Comment:
Do we need to change these? Changing the matrix_name/display_name makes evergreen "loose" the task history so I think we should avoid doing it unnecessarily. For example it makes it harder to see when a task started failing or when it last succeeded.

Reply:
The idea is that we won't need to change again when there are newer RHEL minor versions.  For example, RHEL 7.9 is the only version that will continue to get updates, so we have to migrate.  If we make this change once, we don't lose task history as we have to increment later.

Reply:
SGTM